### ISSUE TITLE
seed

### ISSUE THREAD

**Issue by bridgetli** [oxpig/AntiFold#17]
When you call `get_pdbs_logits` in `main.sample_pdbs`, you set `seed=42` rather than `seed=seed`. And when you call `sample_from_df_logits` in the same function, by not providing a seed, it defaults to 42. When I was running jobs with different seeds, I was getting the same results.

**Comment by Magnushhoie** [comment]
Great point, feel free to submit a PR and I'll merge it :)

**Comment by Magnushhoie** [comment]
Merged in https://github.com/oxpig/AntiFold/pull/18, thanks @bridgetli :)
